### ISSUE TITLE
Compilation fails with webpack on ubuntu server

### ISSUE THREAD

**Issue by marcomd** [FranckFreiburger/vue-pdf#222]
I added vue-pdf to my project, everything went wonderfully on my local until I deployed on the server on which the compilation fails without any message.

In an attempt to solve I updated webpack and node but nothing has changed.

```
$ bundle exec rake assets:precompile RAILS_ENV=pre
yarn install v1.22.4
[1/6] Validating package.json...
[2/6] Resolving packages...
success Already up-to-date.
Done in 0.92s.
Compiling...
Compilation failed:

$
```
Any idea?

local:
- mac os catalina
- node 10.17

server:
- ubuntu 18.04
- node 10.19 then updated to 10.21.1

both:
- yarn 1.22.4
- vue-pdf 4.0.8
- vue 2.6.11
- webpack 4.43 
- backend -> rails 6

**Comment by marcomd** [comment]
I wrote that I have no problems on my local but there are three odd warnings about pdf.js:

WARNING in ./node_modules/pdfjs-dist/build/pdf.js 12060:21-28
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

**Comment by marcomd** [comment]
I somehow resolved it, although I'm still investigating the cause.
I used a sudoers user to connect to the server, then I logged into the usual non-sudoers user, cleaned up the assets and recompiled, so it worked.

`bundle exec rake assets:clobber assets:precompile RAILS_ENV=pre`

After that another fix was needed: in pre production we use a remote bucket, this must have cors enabled.

Now it works, sorry for this issue.
